changes/CHANGELOG-extended-support-cost-projection.md

```diff
@@ -1,11 +1,31 @@
 # What's new in Extended Support Cost Projection
 
-## Extended Support Dashboard - v1.1.0
+## Extended Support Cost Projection - v1.2.0
+
+**Important:** Update to this version requires a forced and recursive update. To update run these commands in your CloudShell (recommended) or other terminal:
+
+```
+python3 -m ensurepip --upgrade
+pip3 install --upgrade cid-cmd
+cid-cmd update --dashboard-id extended-support-cost-projection --force --recursive
+```
+
+You will be presented with the option "proceed and override" to update the RDS and EKS views. Please choose this option to ensure changes to the underlying view queries are applied in your RDS and EKS Extended support views.
```

Comment:
Please add a note that any customisations done to the queries will be overridden, hence customers might want to back-up existing views if they customized them

Reply:
The message has been updated to include the comment mentioning underlying view queries will be overriden and to consider backing them up and re-apply changes after the update takes place should they want to keep them.